src/GraphQL.Analyzers.Tests/FieldNameAnalyzerTests.cs

```diff
@@ -92,12 +122,13 @@ public class MyGraphType : ObjectGraphType
             {
                 public MyGraphType()
                 {
-                    Field<StringGraphType>().Resolve(context => "Test").Name("Text");
+                    {{builderName}}<StringGraphType>().Description("description").Name("Text");
```

Comment:
The `ConnectionBuilder.Resolve` returns void in contrast to `FieldBuilder.Resolve`, which returns the builder instance, so I had to replace it with the `Description` method. Is this inconsistency intended?

Reply:
No, but it would be a breaking change to change the signature in v7.  We can fix v8 (the `develop` branch).